libs/wingsdk/src/target-tf-gcp/app.ts

```diff
@@ -15,6 +16,12 @@ export interface AppProps extends CdktfAppProps {
    */
   readonly projectId: string;
 
+  /**
+   * The Google Cloud region, used for all resources.
+   * @see https://cloud.google.com/functions/docs/locations
+   */
+  readonly region: string;
+
```

Comment:
I just checked we are able to use the same region for both storage and functions, so maybe we can just combine these two properties.